src/main/java/racingcar/domain/RacingRandom.java

```diff
@@ -0,0 +1,15 @@
+package racingcar.domain;
+
+import java.util.Random;
+
+public class RacingRandom {
+    private static final int RANDOM_NUMBER = 10;
+    private static final int MOVABLE_NUMBER = 4;
+    private Random random = new Random();
```

Comment:
```suggestion
    private static final int RANDOM_NUMBER = 10;
    private static final int MOVABLE_NUMBER = 4;
    
    private Random random = new Random();
```
상수와 인스턴스 변수 사이에는 줄 바꿈을 통해서 구분해주세요 🙏

`RANDOM_NUMBER`라는 상수명은 의미가 불명확한 것 같아요 🤔